frontend/appflowy_flutter/lib/workspace/presentation/home/hotkeys.dart

```diff
@@ -25,6 +26,21 @@ class HomeHotKeys extends StatelessWidget {
             .add(const HomeSettingEvent.collapseMenu());
       },
     );
+
+    final HotKey hotKeyForThemeModeChange = HotKey(
+      KeyCode.keyL,
+      modifiers: [
+        Platform.isMacOS ? KeyModifier.meta : KeyModifier.control,
```

Comment:
I think you need to add a `!kIsWeb` check, to not break on Web.

Reply:
How can this break on web?

Reply:
You can't use Platform checks on web

Reply:
Just double-checked, and yes `Platform._operatingSystem` is unsupported on web.

Reply:
I did not see something that prevents these hotkeys to be used on the web. There is also one hotkey for `Ctrl/Meta + \` to hide/show the left menu bar. There also we don't check for web. 
I guess we can not register both of these hotkeys on web, if they do break the web. Or should I not check the platform and just assign Ctrl key as modifier? What do you think is better? @Xazin @LucasXu0

Reply:
I guess this is my oversight, I forgot we don't need to support web platform in the Flutter code on the main project.